### ISSUE TITLE
add factor biplot

### ISSUE THREAD

**Issue by maxheld83** [aiorazabala/qmethod#162]
this is customary for PCA (and factor) analysis in general.

Might also be helpful for inspecting rotations, though it's a bit visual overload #13

**Comment by maxheld83** [comment]
this was originally part of #94 which is now about a simpler plot with *only* the people-variables.

https://github.com/maxheld83/qmethod/commit/d0ecdf80283a1bd9ad527afe862276b24ba6d87c starts development on this.

**Comment by maxheld83** [comment]
like so:

![biplot](https://cloud.githubusercontent.com/assets/5372770/8508109/7aa67af2-2263-11e5-8a00-d48911120475.png)

Notice that the above plot is based on a *weighted* factor scores (as per Brown 1980), so it's not clear if/how this can be interpreted (see #163).

**Comment by maxheld83** [comment]
I'm pretty happy with `q.rotplot()` for now, putting this on the back burner.
Also still seems a little information overload.

**Comment by maxheld83** [comment]
just to document this if I / someone takes this up in future: I *think* **most of the work is done for this**; I have adapted the existing `ggbiplot()` to accept a results object from `qmethod()`, and it seems to work ok (see above).

Caveats that remain:

- [ ] not sure how the weighting and other Q specifics work with a biplot -> #163 
- [ ] `ggbiplot()` is based on SVD, and I have no idea about that, so I cannot be quite sure that my adaptation is correct. Someone must look at this PR -> https://github.com/vqv/ggbiplot/pull/20
- [ ] `ggbiplot()` is *not* on CRAN, so it's unclear how it could be used meaningfully -> https://github.com/vqv/ggbiplot/issues/19
- [ ] `ggbiplot()` would have to be somehow wrapped for `qmethod()`. Guess that's this issue right here.

Bottom line: we're almost there, and no one should start from scratch.
`ggbiplot()` is a great package.

**Comment by maxheld83** [comment]
this might also be relevant, if we're looking for other ways: http://stackoverflow.com/questions/6578355/plotting-pca-biplot-with-ggplot2

**Comment by maxheld83** [comment]
duplicated here https://github.com/QWrks/pensieve/issues/82
